kevm

```diff
@@ -38,7 +38,6 @@ pretty_diff() {
 run_krun() {
     export K_OPTS=-Xss500m
     case "$backend" in
-        ocaml) args=(--interpret) ;;
         *)     args=()            ;;
```

Comment:
This entire `case` statement can go away, and we can remove `${args[@]-}` from the next line.